tools/zuul-projects-checks.py

```diff
@@ -55,6 +56,22 @@ def check_projects_default_branch_with_gh():
     print("\n➜ Checking if Github and local default-branch config is consist")
 
     errors = False
+    gh_token_file = "github_token_file.txt"
+    if os.path.isfile(gh_token_file):
```

Comment:
This is a bit redundant with the FileNotFoundError below. I imagine you can just

Reply:
updated